### ISSUE TITLE
Any plan to release the check-points trained on NYU?

### ISSUE THREAD

**Issue by petertheprocess** [ShngJZ/LightedDepth#3]
incredible work
I was wondering if there are any plans to release the weights associated with the NYU dataset and the training code. Having access to these resources would be extremely beneficial for many researchers and enthusiasts in the field. Could you kindly share any information regarding this or if there's a potential timeline for their release?

**Comment by ShngJZ** [comment]
Yes. I will try to release within this week. Thanks for the reminder!
